storage/src/include/ep-settings.ycp

```diff
@@ -71,8 +77,8 @@
 	    return `item(`id(fs), FileSystems::GetName(fs, "Error"));
 	});
 
-	list<term> partalign_items = maplist(symbol pal, [ `align_optimal, `align_cylinder ], {
-	    return `item(`id(pal), substring( sformat( "%1", pal ), 7 ));
+	list<term> partalign_items = maplist(symbol pal, string label, alignments, {
+	    return `item(`id(pal), label );
```

Comment:
AFAIR the translations are reevaluated and the other translations in the file do it exactly the same way.